.github/workflows/publish-snapshot.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - main
+    tags:
```

Comment:
We don't need this here do we?

Reply:
It's not strictly needed. I added it so that when a new release created we also create the first snapshot for the release after that. Kind of mirrors the behavior that we had before from the "Prepare next development version" commit which would also cause a snapshot to be published

Reply:
Nice!